### PR TITLE
chore(bazel): Bazelify Autocomplete and its dependencies

### PR DESCRIPTION
**Pull request by furtnerl**
### <strong>Pull Request</strong>

<hr>
Hi, thank you for contributing to Barista with this pull request (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please choose the type appropriate for the changes below: <br>

#### Type of PR

<!-- Feature (non-breaking change which adds functionality)

#### Checklist

- [x] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/barista-components/form-field/BUILD.bazel [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+        ":styles",
+        ":src/form-field.html",
+    ],
+    module_name = "@dynatrace/barista-components/form-field",
+    tsconfig = "tsconfig_lib",
+    deps = [
+        "//libs/barista-components/core:compile",
+        "@npm//@angular/core",
+        "@npm//@angular/forms",
+        "@npm//@angular/animations",
```
Comment (ffriedl89) [inline libs/barista-components/form-field/BUILD.bazel:26]:
```suggestion
        "@npm//@angular/core",
        "@npm//@angular/forms",
        "@npm//@angular/animations",
        "@npm//@angular/common",
        "@npm//@angular/cdk",
```

**Comment by lukasholzer** [comment]
Build is failing

**Comment by furtnerl** [comment]
build fixed

File: libs/barista-components/autocomplete/BUILD.bazel [hunk 2]
```diff
@@ -16,7 +43,51 @@ filegroup(
     ]),
 )
 
+sass_binary(
+    name = "styles",
+    src = "src/autocomplete.scss",
+    deps = [],
```
Comment (lukasholzer) [inline libs/barista-components/autocomplete/BUILD.bazel:49]: NIT: 
```suggestion
```

File: libs/barista-components/form-field/BUILD.bazel [hunk 3]
```diff
@@ -16,7 +43,27 @@ filegroup(
...
+    name = "theme",
+    srcs = ["src/_form-field-theme.scss"]
+)
+
+sass_binary(
+    name = "styles",
+    src = "src/form-field.scss",
+    deps = [":theme"],
+)
+
```
Comment (lukasholzer) [inline libs/barista-components/form-field/BUILD.bazel:56]: please add the jest setup as well.

File: libs/barista-components/input/BUILD.bazel [hunk 4]
```diff
@@ -11,18 +35,61 @@ filegroup(
...
+    src = ":src/input.scss",
+    deps = [":theme"],
+)
+
 sass_library(
-    name = "input_scss_lib",
-    srcs = ["src/input.scss"],
+    name = "theme",
+    srcs = [":src/input.scss"],
+    deps = [],
```
Comment (lukasholzer) [inline libs/barista-components/input/BUILD.bazel:53]: NIT:

```suggestion
```

**Pull request merged**